docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+
+To create an export, see [Create exports](/azure/cost-management-billing/costs/tutorial-improved-exports#create-exports).
+
+Here are the high-level steps to create an export:
+
+1. To navigate:
+    1. Sign in to the Azure portal at [https://portal.azure.com](https://portal.azure.com/), search for **Cost Management**.
+    2. Select the required scope and select **Exports** in the left navigation menu.
+    3. Select **+ Create**
+2. On the Basics tab:
+    1. Select the template = **Cost and usage (FOCUS)**
+        :::image type="content" source="./media/create-fabric-workspace-finops/exports-cost-and-usage-focus.png" border="true" alt-text="Screenshot showing creating a new FOCUS export dataset." lightbox="./media/create-fabric-workspace-finops/exports-cost-and-usage-focus.png" :::
+3. On the Datasets tab:
+    1. Fill in **Export prefix** to ensure you have a unique name for the export. For example, *june2024focus*.
+4. On the Destination tab:
+    1. Storage type = Azure blob storage
+    2. Destination and storage = **Use existing**
+    3. Subscription = *Contoso subscription*
+    4. Storage account = *Contoso-storage-account*
+    5. Container = *msexports*
+    6. Directory = focuscost/&lt;*scope-id*&gt;
+    7. Overwrite data = **Enabled**
+5. On the Review + Create tab:
+    - Select **Create**
+
+Ensure that your export completed and that the file is available before moving to the next step.
+
+>[!IMPORTANT]
+> Cost Management exports utilize managed identity. To create an export, you must be an Owner of the storage account or have access to grant permissions to the storage account. To see all required permissions, see [Prerequisites](/azure/cost-management-billing/costs/tutorial-improved-exports#prerequisites).
+
+If you want to automate export creation, consider using the [New-FinOpsCostExport command](https://microsoft.github.io/finops-toolkit/powershell/cost/New-FinOpsCostExport) in the FinOps toolkit PowerShell module.
+
+If you deployed [FinOps hubs](https://aka.ms/finops/hubs), you can use the exports configured against the hub.
+
+## Create a Fabric workspace
+
+1. Sign in to the Fabric app using your [Fabric free account](https://www.microsoft.com/microsoft-fabric/getting-started) or [Microsoft Power BI](https://app.powerbi.com) account.
+2. You can create a new workspace for your FinOps resources or choose to use an existing workspace. To create a workspace, see [Create a workspace](/fabric/get-started/create-workspaces).
+
+If you already have a [Microsoft Fabric workspace](/fabric/get-started/workspaces), you can skip creating a new workspace, unless you want to keep your FinOps resources separate from others.
+
+If you didn't set up Microsoft Fabric, see [Enable Microsoft Fabric for your organization](/fabric/admin/fabric-switch).
```

Comment:
Should we make this the first step to enable Fabric if it hasn't been enabled?

Reply:
yep. moved